synapse/lib/cell.py

```diff
@@ -774,6 +774,11 @@ class Cell(s_nexus.Pusher, s_telepath.Aware):
             'type': 'string',
             'hideconf': True,
         },
+        'cell:minspace': {
+            'default': 5,
+            'description': 'Minimum free space percentage before setting the cell read-only.',
+            'type': 'integer',
```

Comment:
We can specify a range here https://json-schema.org/understanding-json-schema/reference/numeric.html#range